brambling/forms/organizer.py

```diff
@@ -265,6 +266,30 @@ def _post_clean(self):
             self._update_errors(e)
 
 
+class CustomFormForm(forms.ModelForm):
+    class Meta:
+        model = CustomForm
+        exclude = ('event',)
```

Comment:
I think so? Why would I ever change that as an event organizer under the current behavior?